Frontend/package.json

```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "axios": "^0.18.0",
+    "puppeteer": "^1.13.0",
```

Comment:
This shouldn't be added to the frontend repo. It would be better to make a separate repo for testing the frontend.

If you do decide to add it to this repo regardless, add this to the dev dependencies rather then the app dependencies.